pkg/volume/csi/csi_mounter.go

```diff
@@ -195,7 +195,12 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 	if len(fsType) == 0 {
 		fsType = defaultFSType
 	}
-
+	nodePublishCredentials := map[string]string{}
```

Comment:
Not your code, but, can you add a commit to remove probe related code in this PR too:

```go
...
 	if err := csi.NodeProbe(ctx, csiVersion); err != nil {
...
```
and all related comments, e.g.:
```go
// TODO (vladimirvivien) use this method to probe controller using CSI.NodeProbe() call
```

Reply:
done